contracts/staking/contracts/src/immutable/MixinStorage.sol

```diff
@@ -69,12 +69,10 @@ contract MixinStorage is
     mapping (address => uint256) internal _withdrawableStakeByOwner;
 
     // tracking Pool Id
-    bytes32 public nextPoolId = INITIAL_POOL_ID;
+    bytes32 public nextPoolId = bytes32(uint256(1));
```

Comment:
I think what would be a bit cleaner is tracking `lastPoolId` so that it can default to 0. Then when creating a pool, we calculate the new `poolId`, create the pool, and update `lastPoolId`.